sdk/keyvault/keyvault-secrets/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
```

Comment:
Minor nit: I’d probably use the word “Fixes” to indicate that something was broken so users know if updating will fix their issue, but otherwise looks good!